### PR TITLE
v2v: add case on deleting warning info on rhev-apt

### PR DESCRIPTION
**Pull request by vwu-vera**
Check bz #1945549 and case RHEL-254337

Signed-off-by: vwu-vera <vwu@redhat.com>

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.rhev_apt.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (617.96 s)

 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.rhev_apt.esx_67.rhev.rhv_upload.it_vddk.esx_uri: PASS (792.01 s)

 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.rhev_apt.esx_67.rhev.rhv_upload.it_default: PASS (3558.09 s)

 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.rhev_apt.esx_67.rhev.rhv.it_vddk.vpx_uri: PASS (975.77 s)

 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.rhev_apt.esx_67.rhev.rhv.it_vddk.esx_uri: PASS (1007.80 s)

 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.rhev_apt.esx_67.rhev.rhv.it_default: PASS (3826.11 s)

**Comment by xiaodwan** [comment]
You don't have to test all combination of output to rhv server, because v2v handles sources and destinations independently.
If one output method is passed, then others should also be fine.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -102,6 +102,14 @@
...
                     expect_msg = 'no'
                     msg_content = 'virt-v2v: warning: /usr/share/virt-tools/pnp_wait.exe is missing.*?Firstboot scripts may conflict with PnP'
+                - rhev_apt:
+                    only esx_67
+                    no libvirt
+                    checkpoint = 'rhev_file'
+                    os_version = OS_VERSION_RHEV_FILE_V2V_EXAMPLE
+                    main_vm = RHEV_FILE_VM_NAME_V2V_EXAMPLE
+                    expect_msg = 'no'
+                    msg_content = 'virt-v2v: warning: /usr/share/virt-tools/rhev-apt.exe is missing'
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:112]: This message check can be merged into other windows cases, it's a waist of time to only add a case to check such non-existed messages. The scripts will grow larger and larger, but we can't get more benefit from it.
And the compatibility with rhel8 should also be considered.

**Comment by vwu-vera** [comment]
Modify the TC according to the reviewer.

Run result passed:
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win2019.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (666.98 s)

@xiaodwan  Please help to review the commit. Thanks.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 2]
```diff
@@ -141,6 +141,9 @@
                         - win2019:
                             main_vm = VM_NAME_WIN2019_V2V_EXAMPLE
                             os_version = "win2019"
+                            checkpoint = 'rhev_file'
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:144]: First, you can't mix different cases with same checkpoint even it works in logic, but that will break the maintainability of the whole project. so please don't do that unless the cases are different scenarios of the checkpoint.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 3]
```diff
@@ -141,6 +141,9 @@
...
+                            expect_msg = 'no'
+                            msg_content = 'virt-v2v: warning: /usr/share/virt-tools/rhev-apt.exe is missing'
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:146]: Multiple messages can be put into msg_content, they are separated by '%'.
see https://github.com/avocado-framework/avocado-vt/blob/273696156004e763bb471866ad576bed9898944e/virttest/utils_v2v.py#L1513.

**Comment by vwu-vera** [comment]
Modify the TC according to the suggestion.

Run result passed:
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.rhev_file.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (551.34 s)

check the log:
2021-08-03 01:23:36,041 utils_v2v        L1493 INFO | Searching for not expected log: virt-v2v: warning: /usr/share/virt-tools/rhev-apt.exe is missing
2021-08-03 01:23:36,041 utils_v2v        L1501 INFO | Not find log: virt-v2v: warning: /usr/share/virt-tools/rhev-apt.exe is missing

@xiaodwan Please help to review the latest commit. Thanks.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**